packages/scheduler/src/queueMicrotask.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+function queueMicrotaskPolyfill(callback: boolean => void) {
+  if (typeof queueMicrotask !== 'undefined') {
+    queueMicrotask(callback);
+  } else if (typeof Promise !== 'undefined') {
+    Promise.resolve(null)
+      .then(callback)
+      .catch(e => {
+        setTimeout(() => {
```

Comment:
Important note: when we fallback to the promise implementation, we have to push errors into a new task or they'll be treated as unhandled promise rejections. I'm using the MDN polyfill version here, but we may want to schedule a MicrotaskPriority task instead, since these can be pushed behind other work.

Reply:
Not too concerned since it really only affects debugging and all modern browsers support `queueMicrotask`. (God I hate this problem so much, though.)